### ISSUE TITLE
Feature request - Assemblies

### ISSUE THREAD

**Issue by gumyr** [meadiode/cq_gears#5]
Thank you for building this. I was going to port my OpenSCAD [gear_warehouse](https://www.thingiverse.com/thing:4603821) to my CadQuery [cq_warehouse](https://github.com/gumyr/cq_warehouse) but fortunately found your work instead.

As an enhancement would you consider creating CadQuery Assemblies in addition to the Compound object gear sets? Assemblies seem like a natural fit and port well to other CAD systems via STEP files.

**Comment by meadiode** [comment]
Yep, seems like a good idea, I'll add 'assembly' methods also to the gear-set classes.
